mat64/dense.go

```diff
@@ -174,6 +193,8 @@ func (m *Dense) SetRow(i int, src []float64) int {
 	return min(len(src), m.mat.Cols)
 }
 
+// RowView returns a slice of float64 reflecting a row that is backed by the matrix data.
```

Comment:
The behaviour changed. This method now returns a *Vector, so s/[]float64/*Vector/.